app/experimenter/nimbus-ui/src/components/GQLErrorAlert/mocks.tsx

```diff
@@ -0,0 +1,72 @@
+  },
+  message:
+    "JSON.parse: unexpected character at line 1 column 1 of the JSON data",
+};
+
+export const NETWORK_ERROR = {
+  graphQLErrors: [],
+  networkError: {
+    name: "Error",
```

Comment:
I may have been able to check for this `name` string instead of the [type guards functions](https://github.com/mozilla/experimenter/pull/6008/files#diff-ab35b3e1a6dc836a528565f1ed49e372648c894bb4ab2164fda7b4c92bfb13abR13-R29), but it seems like we wanted this to handle regular `Error` instances too and the typeguards made TS happy.